### PR TITLE
Finetuned default index

### PR DESCRIPTION
**Pull request by ro0NL**
Any thoughts?

Before:
image

image

After:
image

image

### CODE REVIEW

**Comment by ro0NL** [comment]
@javiereguiluz wdyt?

File: app/Resources/views/default/index.html.twig [hunk 1]
```diff
@@ -50,22 +50,25 @@
 <style>
     body { background: #F5F5F5; font: 18px/1.5 sans-serif; }
     h1, h2 { line-height: 1.2; margin: 0 0 .5em; }
-    h1 { font-size: 36px; }
+    h1 { font-size: 26px; }
```
Comment (javiereguiluz) [inline app/Resources/views/default/index.html.twig]: This value can only be `24` or `28` (we use the traditional typographic scale: 16, 18, 21, 24, 28, 32, 36, 48, 64, 72, 96, etc.)

**Comment by javiereguiluz** [comment]
To be completely honest, I don't like the two main changes proposed in this PR: removing the Welcome message from the main content section and fading some texts in a light gray 😢

**Comment by fabpot** [comment]
@ro0NL We worked a lot on those designs. So, except if there are things that need to be fixed, I don't see the need to change them.

**Comment by ro0NL** [comment]
Your call :) i was just playing around with contrast. Imo it looks better, but this matter of taste. Im not a designer ;-)

**Pull request closed without merging**